### PR TITLE
[release] change tag to main

### PR DESCRIPTION
**Pull request by MarcelKoch**

### CODE REVIEW

**Comment by yhmtsai** [comment]
Some readme link should use main not develop

**Review by yhmtsai — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
-|:-:|:-:|:-:|:-:|:-:|:-:|
+[![Build status](https://gitlab.com/ginkgo-project/ginkgo-public-ci/badges/main/pipeline.svg)](https://gitlab.com/ginkgo-project/ginkgo-public-ci/-/pipelines?page=1&scope=branches&ref=main)|[![OSX-build](https://github.com/ginkgo-project/ginkgo/actions/workflows/osx.yml/badge.svg)](https://github.com/ginkgo-project/ginkgo/actions/workflows/osx.yml)| [![codecov](https://codecov.io/gh/ginkgo-project/ginkgo/branch/main/graph/badge.svg)](https://codecov.io/gh/ginkgo-project/ginkgo) |[![Maintainability Rating](https://sonarcloud.io/api/project_badges/measure?project=ginkgo-project_ginkgo&metric=sqale_rating)](https://sonarcloud.io/dashboard?id=ginkgo-project_ginkgo)|[![Reliability Rating](https://sonarcloud.io/api/project_badges/measure?project=ginkgo-project_ginkgo&metric=reliability_rating)](https://sonarcloud.io/dashboard?id=ginkgo-project_ginkgo)|[![CDash dashboard](https://img.shields.io/badge/CDash-Access-blue.svg)](https://my.cdash.org/index.php?project=Ginkgo+Project)
+|:-:|:-:|:----------------------------------------------------------------------------------------------------------------------------------:|:-:|:-:|:-:|
```
Comment (yhmtsai) [inline README.md:10]: any reason for additional dash here?

**Review by pratikvn — APPROVED** [review]
LGTM!

File: README.md (hunk 1 above)
Comment (MarcelKoch) [reply README.md]: autoformatting. I have removed it

**Pull request merged**